changelogs/fragments/266-default-database-for-mysql-user.yml

```diff
@@ -0,0 +1,2 @@
+minor_change:
```

Comment:
```suggestion
minor_changes:
```
this should fix sanity tests

Reply:
*facepalm* thanks, i was wondering whats wrong...

Reply:
you're welcome!